core/src/main/java/com/example/AutomatedReviewTest.java

```diff
@@ -0,0 +1,36 @@
+package com.example;
+
+import java.util.*;
+
+// The goal of this code is to find out if code analysis will automatically add
+// commentary to a pull request when it finds problems with the code.
+
+public abstract class AutomatedReviewTest extends Object {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this abstract class name to match the regular expression '^Abstract[A-Z][a-zA-Z0-9]*$'. ([squid:S00118](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AS00118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack&pullRequest=9&issues=AWY_zChlvOM6zLh9Plcx&open=AWY_zChlvOM6zLh9Plcx)

[ISSUE_KEY]: AWY_zChlvOM6zLh9Plcx